src/OpenSage.Game/Logic/PlayerSettings.cs

```diff
@@ -21,14 +21,16 @@ public struct PlayerSetting
         public PlayerTemplate Template { get; set; }
         public PlayerOwner Owner { get; set; }
         public string Name { get; set; }
+        public byte Team { get; set; }
 
-        public PlayerSetting(int? startPosition, PlayerTemplate template, ColorRgb color, PlayerOwner owner = PlayerOwner.None, string name = "")
+        public PlayerSetting(int? startPosition, PlayerTemplate template, ColorRgb color, byte team, PlayerOwner owner = PlayerOwner.None, string name = "")
```

Comment:
We usually don't use **byte** as a numeric type. Just use **int** here, since it makes no big difference and avoids any future limitations

Reply:
Sure thing. I swear there was a reason I used byte (I thought something else was already configured with it?) but skimming the code now I can't figure out what it was. Anyway, I agree int is the reasonable solution here.